packages/terms-of-service/src/class-terms-of-service.php

```diff
@@ -53,18 +52,25 @@ public function reject() {
 	 * Returns whether the master user has agreed to the terms of service.
 	 *
 	 * The following conditions have to be met in order to agree to the terms of service.
-	 * 1. The master user has gone though the connect flow.
+	 * 1. The master user has gone though the connect flow (deprecated @since 8.9.0).
```

Comment:
This part is not deprecated, it means that the user has started the flow and went down the funnel towards the connection. In this case we mean that we're checking whether the local option is set to `true`.

Reply:
Thanks @zinigor ! Fixed :)